x-pack/qa/rolling-upgrade-multi-cluster/src/test/java/org/elasticsearch/upgrades/CcrRollingUpgradeIT.java

```diff
@@ -224,8 +226,20 @@ public void testCannotFollowLeaderInUpgradedCluster() throws Exception {
                 ResponseException.class,
                 () -> followIndex(leaderClient(), "follower", "not_supported", "not_supported")
             );
-            assertThat(e.getMessage(), containsString("the snapshot was created with index version ["));
-            assertThat(e.getMessage(), containsString("] which is higher than the version used by this node ["));
+
+            assertThat(
+                e.getMessage(),
+                allOf(
```

Comment:
I think this should be `anyOf(allOf(...), allOf(...))` - we don't have all possible combinations of messages